app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -904,6 +918,115 @@ public boolean onOptionsItemSelected(MenuItem item) {
     return super.onOptionsItemSelected(item);
   }
 
+  /** Dialog to take user confirmation before deleting all notes */
+  private void showClearAllNotesDialog() {
+
+    AlertDialog.Builder builder;
+    if (sharedPreferenceUtil != null && sharedPreferenceUtil.nightMode()) { // Night Mode support
+      builder = new AlertDialog.Builder(this, R.style.AppTheme_Dialog_Night);
+    } else {
+      builder = new AlertDialog.Builder(this);
+    }
+
+    builder.setMessage(this.getString(R.string.delete_notes_confirmation_msg))
+        .setNeutralButton(this.getString(R.string.dismiss), new DialogInterface.OnClickListener() {
```

Comment:
I am reasonably sure you can just pass in a null if your dialogClickListener is doing nothing

Reply:
Also NeutralButton is the wrong method I think, use setNegativeButton

Reply:
Refactored and changed to setNegativeButton() [everywhere](https://github.com/kiwix/kiwix-android/commit/873eb2bce86c76b11962367701cb66bdf66b9f5e)